advanced/sr_gui_movement_recorder/src/sr_gui_movement_recorder/movement_recorder.py

```diff
@@ -303,13 +304,15 @@ def __init__(self, context):
 
         # setting up the hand selection
         hand_finder = HandFinder()
-        self.hand_parameters = hand_finder.get_hand_parameters()
+        if hand_finder.hand_e_available():
```

Comment:
although probably move line 311 (```self.hand_combo_box = QComboBox()```) to before the ```if``` because things might break if it doesn't exist at all. it can just be there but not added to the layout and therefore not visible.